indexers/vector/PysparnnIndexer/tests/test_pysparnnindexer.py

```diff
@@ -100,13 +106,21 @@ def test_add_query_add_with_closing(indexer, features):
     query_results = indexer.query(vectors=features[:5], top_k=1)
     indexer.close()
     indexer_query = PysparnnIndexer(metas=indexer.metas, metric='cosine')
-    #  ValueError: Not possible query while indexing
     indexer_query.add(keys=list(range(0, 50)), vectors=features)
     indexer_query_results = indexer_query.query(vectors=features[:5], top_k=1)
 
     assert (query_results[0] == indexer_query_results[0]).all()    
     assert (query_results[1] == indexer_query_results[1]).all()
 
+
+def test_add_close_add_newkeys(indexer, features):
```

Comment:
can u add some examples like this that also leverage `update` and `delete` API?

Reply:
done